e2e/utils.go

```diff
@@ -498,3 +498,17 @@ func oneReplicaDeployYaml(template string) string {
 	var re = regexp.MustCompile(`(\s+replicas:) \d+`)
 	return re.ReplaceAllString(template, `$1 1`)
 }
+
+// nolint:unparam // currently major is always 1, this can change the the future
+func k8sVersionGreaterEquals(c kubernetes.Interface, major, minor int) bool {
```

Comment:
Naming looks a bit odd to me, a few suggestions:

checkK8sVersionMatchRequired()
checkK8sVersionMeetRequired()
isK8sVersionMatchRequired()
isK8sVersionMeetRequired()

Reply:
I took the naming from `-ge` (Greater Equals, Bash). `Match` is not correct, as it needs `>=` (pronounced: greater equals). Not sure about `Meet`...